test/integration/instance_with_attached_disk/controls/www.rb

```diff
@@ -18,8 +18,25 @@
 control "www" do
   title "WWW Access"
 
-  describe http("http://#{http_address}:#{port}/", method: 'GET', open_timeout: 30, read_timeout: 30) do
-    its('status') { should eq 200 }
-    its('body') { should include 'Hello, world!' }
+  describe "HTTP service" do
+    let(:resource) do
+      http_resource = -> { http("http://#{http_address}:#{port}/", method: 'GET', open_timeout: 30, read_timeout: 30) }
+      Timeout::timeout(400) do
+        r = http_resource.call
+        until r.status == 200
```

Comment:
Will this put the tests in to an infinite loop if the status is never 200?

Reply:
No -- after 400 seconds (the amount of time we were `sleep`ing before running tests at all previously), it will raise a `Timeout::Error`

Reply:
Some how I missed the very apparent `Timeout` on line 24. 😊